docs/03-tutorials/00-api-exposure/apix-03-expose-multiple-services.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Expose multiple services on the same host
+---
+
+This tutorial shows how to expose a workload with multiple services that share the same host on different paths.
+
+You can either define a service at the root level, which is applied to all paths except the ones you've explicitly set service for at the rules level, or you can just define different services on each path separately without the need to define a root service.
```

Comment:
I just noticed, this is the reversed order compared to how it comes up in the following text. Please adjust it here or switch the sections so that the order matches :)